apiml-security-common/src/main/java/org/zowe/apiml/security/common/filter/StoreAccessTokenInfoFilter.java

```diff
@@ -12,50 +12,63 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectReader;
-import lombok.RequiredArgsConstructor;
-import org.springframework.web.filter.OncePerRequestFilter;
-import org.zowe.apiml.security.common.handler.SuccessfulAccessTokenHandler;
-import org.zowe.apiml.security.common.error.AccessTokenBodyNotValidException;
-import org.zowe.apiml.security.common.error.AuthExceptionHandler;
-
 import jakarta.servlet.FilterChain;
 import jakarta.servlet.ServletException;
 import jakarta.servlet.ServletInputStream;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.filter.OncePerRequestFilter;
+import org.zowe.apiml.message.log.ApimlLogger;
+import org.zowe.apiml.product.logging.annotations.InjectApimlLogger;
+import org.zowe.apiml.security.common.error.AccessTokenBodyNotValidException;
+import org.zowe.apiml.security.common.error.AuthExceptionHandler;
+import org.zowe.apiml.security.common.handler.ServletErrorUtils;
+import org.zowe.apiml.security.common.handler.SuccessfulAccessTokenHandler;
+
 import java.io.IOException;
 import java.util.Set;
+import java.util.function.BiConsumer;
 import java.util.stream.Collectors;
 
+//todo fix me for non-modulith
```

Comment:
Is this TODO valid? If it is, it should be in upper case so IDEs consider it as one